src/App.jsx

```diff
@@ -14,33 +17,94 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+const SORT_BY_ALPHABET = 'alphabet';
+const SORT_BY_LENGTH = 'length';
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+function prepareGoods(goods, { sortField }, isReversed) {
```

Comment:
The method name 'prepareGoods' should start with a verb according to the checklist item: "Method's name should start with a verb".